API/Startup.cs

```diff
@@ -67,10 +71,12 @@ public void ConfigureServices(IServiceCollection services)
                     memoryCacheOptions.CompactionPercentage = cacheSettings.MemoryCacheCompactionPercentage;
                 })
                 .AddLazyCache()
+                .AddScoped<ISpaceRepository, SpaceRepository>()
                 .AddSingleton<ICustomerRepository, DapperCustomerRepository>()
                 .AddScoped<IEntityCounterRepository, EntityCounterRepository>()
                 .AddSingleton<IAuthServicesRepository, DapperAuthServicesRepository>()
                 .AddScoped<ICustomerQueueRepository, CustomerQueueRepository>()
+                .AddScoped<IAssetRepository, DapperAssetRepository>()
                 .ConfigureMediatR()
                 .ConfigureSwagger()

```

Comment:
Not specifically related to this PR but we should take note to update XML comments etc for better swagger docs.